assets/src/edit-story/panels/elements/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as ActionButton } from './actionButton';
```

Comment:
The naming of this directory is strange here. Elements are the name of our blocks in the editor. This directory should be named components or form elements.

Reply:
Fixed in d68854468.